apps/leo_gateway/test/leo_nfs_integration_tests.sh

```diff
@@ -110,31 +116,47 @@ function make_many_files() {
     return 0
 }
 
+function check_dummy() {
+    if [ ! -d $DST_OLD_DIR ]; then
+        echo "[Failed] Old Dummy Dir File Missing"
+        return 1
+    elif [ ! -d $DST_NEW_DIR ]; then
+        echo "[Failed] NEW Dummy Dir File Missing"
+        return 1
+    elif [ ! -d $DST_BOTH_DIR ]; then
+        echo "[Failed] OLD+NEW Dummy Dir File Missing"
+        return 1
+    fi
+    return 0
+}
+
 # Tests
 {
     # try block
     eval $CMD_MOUNT &&
     eval $CMD_DF &&
+    check_dummy &&
     eval $CMD_MKDIR &&
     eval $CMD_CP_1K &&
     eval $CMD_CP_1M &&
     eval $CMD_CP_50M &&
     eval $CMD_DIFF_1K &&
     eval $CMD_DIFF_1M &&
     eval $CMD_DIFF_50M &&
-    ls_validate_num_of_childs $MOUNT_DIR 4 && 
+    ls_validate_num_of_childs $MOUNT_DIR 7 && 
     eval $CMD_RM_1K &&
-    ls_validate_num_of_childs $MOUNT_DIR 3 &&
+    ls_validate_num_of_childs $MOUNT_DIR 6 &&
     eval $CMD_MV_50M &&
-    ls_validate_num_of_childs $MOUNT_DIR 2 &&
+    sleep 5 &&
+    ls_validate_num_of_childs $MOUNT_DIR 5 &&
     find_validate_name_exp $MOUNT_DIR $FNAME_50M $DST_50M_FILE2 &&
     eval $CMD_RM_50M &&
     eval $CMD_RMDIR_SUB &&
     eval $CMD_RM_SUB_ROOT &&
-    ls_validate_num_of_childs $MOUNT_DIR 1 &&
+    ls_validate_num_of_childs $MOUNT_DIR 4 &&
     eval $CMD_TOUCH &&
     stat_validate_size $DST_1M_FILE 1048576 &&
-    du_validate_size_in_mbyte $MOUNT_DIR 2 &&
+    du_validate_size_in_mbyte $MOUNT_DIR 5 &&
```

Comment:
Test has failed on my env at the above line.
`2` instead of `5` looks appropriate for the 2nd arugement.

Also please check `du_validate_size_in_mbyte` implementation.
I got the following output.
```
[Failed]du_validate_size_in_mbyte expected:1
1
1
2 value:5
```
since some directories for testing dummy dir have been added into the mount dir,
you need to change the option to `du` to output only the value summed up.